examples/protocols/sntp/main/sntp_example_main.c

```diff
@@ -157,10 +166,43 @@ static void initialize_sntp(void)
 {
     ESP_LOGI(TAG, "Initializing SNTP");
     sntp_setoperatingmode(SNTP_OPMODE_POLL);
+
+/*
+ * If 'NTP over DHCP' is enabled, we set dynamic pool address
+ * as a 'secondary' server. It will act as a fallback server in case that address
+ * provided via NTP over DHCP is not accessible
+ */
+#if defined LWIP_DHCP_GET_NTP_SRV && (SNTP_MAX_SERVERS > 1)
```

Comment:
```suggestion
#if  (LWIP_DHCP_GET_NTP_SRV == 1) && (SNTP_MAX_SERVERS > 1)
```
or
```cpp
#if defined CONFIG_LWIP_DHCP_GET_NTP_SRV
```

Otherwise, if the DHCP-NTP option is not set (default), this condition is true because after:

https://github.com/espressif/esp-idf/blob/4a011f318377ab717b9dc459019c1055602c93ff/components/lwip/port/esp32/include/lwipopts.h#L1065-L1067

`LWIP_DHCP_GET_NTP_SRV` is undefined, but

https://github.com/espressif/esp-lwip/blob/2749568fe15df2003f6c3f37f0dfd44f8f01fcd6/src/include/lwip/opt.h#L956-L958

sets this to `0`, so it's actually defined